tox.ini

```diff
@@ -1,48 +1,26 @@
 [tox]
-envlist =
-    py{37,38,39,310}-dj{22,30,31,32,40}
-    flake8
-    isort
-    black
+envlist = py{38,39,310}-dj{22,30,31,32,40}
+skip_missing_interpreters = True
 isolated_build = True
 basepython = python3
 
+[gh-actions]
+python =
+    3.8: python3.8
+    3.9: python3.9
+    3.10: python3.10
+
 [testenv]
+whitelist_externals = pytest
 setenv =
     PYTHONPATH = {toxinidir}
     DJANGO_SETTINGS_MODULE = tests.settings
 deps =
-    dj22: Django>=2.2,<2.3
-    dj30: Django>=3.0,<3.1
-    dj31: Django>=3.1,<3.2
-    dj32: Django>=3.2,<3.3
     dj40: Django>=4.0,<4.1
-
-install_command = pip install -U {opts} {packages}
-commands =
-    ; Install any dependencies that have changed since this
-    ; tox env was created. Installing in editable mode because it is faster.
-    pip install -e .
-    coverage run --source="{toxinidir}/tbxforms" -m django test {posargs:tbxforms}
-    django-admin collectstatic --noinput
-    django-admin check
-    django-admin makemigrations --check --noinput
-    coverage report -m --omit="{toxinidir}/tbxforms/tests/*"
-extras = testing
-
-[testenv:black]
-basepython = python3
-commands =
-    black --check ./
-
-[testenv:flake8]
-basepython = python3
-commands =
-    flake8 tbxforms
-
-[testenv:isort]
-basepython = python3
-usedevelop = false
-changedir = {toxinidir}
+    dj32: Django>=3.2,<4.0
+    dj31: Django>=3.1,<3.2
+    dj30: Django>=3.0,<3.1
+    dj22: Django>=2.2,<2.3
```

Comment:
Side note: Django < 3.2 are fully deprecated
in fact, the only supported versions are 3.2, 4.1 and 4.2 - https://endoflife.date/django